frontend/src/Requirements.js

```diff
@@ -82,7 +82,7 @@ export function populateReqTree(reqTree){
                   return(<li key={index} className='settled' onClick={(e)=>{toggleSettle(course, requirement['path_to'], false)}}>{course}</li>);
                 })}
                 {requirement['unsettled'].map((course, index)=>{
-                  return(<li key={index} className='unsettled text-muted' onClick={(e)=>{toggleSettle(course, requirement['path_to'], true)}}>{course}</li>);
+                  return(<li key={index} className='unsettled text-muted' onClick={(e)=>{toggleSettle(course, requirement['path_to'], true)}}>{course} <i class="fa fa-exclamation-circle" title="This course could satisify multiple requirements. Click to settle it here."></i></li>);
```

Comment:
You have to hover over the icon and leave your mouse there for a little bit before the text pops up. Do you think we should have it pop up immediately on hover?

Reply:
Also, it should be `className`, not `class`.

Reply:
Switched to `className`